server/fishtest/stats/stat_util.py

```diff
@@ -251,7 +246,7 @@ def update_SPRT(R, sprt):
 
     R_ = R.get("pentanomial", R3)
 
-    batch_size = sprt.get("batch_size", 1)
+    batch_size = sprt.get["batch_size"]
```

Comment:
I-m wondering if this should be:
`batch_size = sprt["batch_size"]`

```
Jan 09 11:14:53 vps1572612706 pserve[29311]:   File "/home/usr00/fishtest/server/fishtest/api.py", line 378, in update_task
Jan 09 11:14:53 vps1572612706 pserve[29311]:     result = self.request.rundb.update_task(
Jan 09 11:14:53 vps1572612706 pserve[29311]:   File "/home/usr00/fishtest/server/fishtest/rundb.py", line 871, in update_task
Jan 09 11:14:53 vps1572612706 pserve[29311]:     return self.sync_update_task(worker_info, run_id, task_id, stats, spsa)
Jan 09 11:14:53 vps1572612706 pserve[29311]:   File "/home/usr00/fishtest/server/fishtest/rundb.py", line 961, in sync_update_task
Jan 09 11:14:53 vps1572612706 pserve[29311]:     fishtest.stats.stat_util.update_SPRT(updated_results, sprt)
Jan 09 11:14:53 vps1572612706 pserve[29311]:   File "/home/usr00/fishtest/server/fishtest/stats/stat_util.py", line 249, in update_SPRT
Jan 09 11:14:53 vps1572612706 pserve[29311]:     batch_size = sprt.get["batch_size"]

```

Reply:
Yep.... Dyslexia.